lib/index.js

```diff
@@ -100,35 +114,24 @@ function setupRateLimitEventExt(server, options) {
   };
 
   server.ext(event, (request, reply) => {
+    // This handler is going to be called one time per registration of patova
     // If current response is already not conformant we can stop
-    const currentRlRequest = store.get(request.id);
-    if (currentRlRequest && !currentRlRequest.isConformant()) { return; }
+    const currentRequestLimit = request.plugins.patova && request.plugins.patova.limit
+    if (currentRequestLimit && !currentRequestLimit.conformant) { return; }
```

Comment:
There is a little tradeoff on that, in theory, any plugin could change the response from being an error to being something else, I think it would be weird (and confusing) but in theory it could happen; I'm ok on telling that in such cases we don't want to apply the limits but it is a tradeoff to keep in mind